src/state/slices/swapSlice/types.ts

```diff
@@ -0,0 +1,7 @@
+import type { Swap } from '@shapeshiftoss/swapper'
+
+export type SwapState = {
+  swaps: Swap[]
+}
```

Comment:
See https://github.com/shapeshift/web/pull/9625/files#r2114195967 - we probably want to index by Id and have a collection of IDs, both which will help  iterations and lookups (`O(1)`).
Then if we need to have an ordered list of sort to iterate over, we could have a selector that does that, or perhaps also have a `swaps` in addition to the byId

So something like

```typescript
export type SwapState = {
 byId: Record<string, Swap>
 ids: string[]
 swaps: Swap[] // maybe, maybe not, probably better to have a reducer for this one
}
```

WDYT?

Reply:
Why not! Happy with byId and ids as it simplify things! in [fbcd366](https://github.com/shapeshift/web/pull/9625/commits/fbcd366ded019ee7c8586022801e2a9625cc80dd)